admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,101 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes
+when scheduling pods using labels and node selectors, as well as the node
+blacklist setting.  
+
+== Constraining Pod Placement
+
+Adding a label to the `nodeSelectorLabelBlacklist` section of a master
+configuration file prevents pods from being created if their labels match any
+values in the `nodeSelectorLabelBlacklist`. This means, if no node selector is
+available, then the scheduler can still place a pod on a node with a blacklisted
+label.
+
+For example, if you have a cluster consisting of five datacentres spread across
+two regions: say, in the US, "us-east", "us-central", and "us-west"; and in
+APAC, "apac-east" and "apac-west". Each node in each geographical region
+link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[should be
+labeled accordingly]. For example, `region: us-east`.
+
+As a cluster administrator, you can create an infrastructure where application
+developers are deploying pods onto the nodes closest to their geographical
+location. You can
+link:../admin_guide/managing_projects.html#using-node-selectors[create a node
+selector], grouping the US datacentres into `superregion: us` and the APAC
+datacentres into `superregion: apac`.
+
+To prevent developers from overloading a particular region, you can add the
```

Comment:
"overloading" might not be the best term here, especially since the scheduler should generally prevent that from happening. I think rephrasing this to something along the lines of "To maintain even loading of resources per-datacenter, you can add..." This encapsulates both the abuse-prevention use case (implied by the term "overloading") and the cost-management use case, which is what I was basing my original example upon.